app/src/main/java/org/fossasia/susi/ai/MainActivity.java

```diff
@@ -119,7 +120,10 @@ public boolean onOptionsItemSelected(MenuItem item) {
     @Override
     protected void onResume() {
         super.onResume();
-        etMessage.requestFocus();
+        //etMessage.requestFocus();
```

Comment:
Technically the user won't want the keyboard to popup as he enters the chat app. (As it is in the case of Whatsapp or Allo)